resources/views/fl-builder-archive.blade.php

```diff
@@ -0,0 +1,13 @@
+@extends("ItinerisSageFLBuilderViews::layouts.archive")
+
+@section('content')
+    @php
+    $ids = FLThemeBuilderLayoutData::get_current_page_content_ids();
+    if ( 'fl-theme-layout' == get_post_type() && count( $ids ) > 1 ) {
+        $post_id = FLBuilderModel::get_post_id();
+    } else {
+        $post_id = $ids[0];
+    }
+    FLBuilder::render_content_by_id($post_id, 'div', apply_filters('fl_theme_builder_content_attrs', []));
+    @endphp
+@endsection
```

Comment:
```suggestion
@extends("ItinerisSageFLBuilderViews::layouts.archive")

@section('content')
    @php
    $ids = FLThemeBuilderLayoutData::get_current_page_content_ids();

    if ('fl-theme-layout' === get_post_type() && count($ids) > 1) {
        $post_id = FLBuilderModel::get_post_id();
    } else {
        $post_id = $ids[0] ?? get_the_ID();
    }

    FLBuilder::render_content_by_id($post_id, 'div', apply_filters('fl_theme_builder_content_attrs', []));
    @endphp
@endsection
```

Also, is there any way you can pass `$post_id` to the template instead?

```suggestion
@extends("ItinerisSageFLBuilderViews::layouts.archive")

@section('content')
    @php
    FLBuilder::render_content_by_id($post_id, 'div', apply_filters('fl_theme_builder_content_attrs', []));
    @endphp
@endsection
```

Reply:
@codepuncher , unfortunately not, `makeLoader()` do not pass variables, for some reason.